src/base/display.jl

```diff
@@ -1,13 +1,20 @@
-function Base.show(io::IO, drv::Driver)
-    drv.ptr == C_NULL && (return print(io, "NULL Driver"))
+function Base.show(io::IO, drv::Driver)::IO
+    if drv.ptr == C_NULL
+        print(io, "NULL Driver")
+        return io
+    end
```

Comment:
This could be an `else`, makes the return single as well.

Reply:
https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/aecd4d89c1ef7e892e7e953001ff71913be18b24